todo_app/app.py

```diff
@@ -1,17 +1,26 @@
 from flask import Flask, render_template, request, redirect, url_for, abort
 from flask_login import LoginManager, login_required
-import flask_login
+import flask_login, logging
 from flask_login.mixins import UserMixin
 from oauthlib.oauth2 import WebApplicationClient
 import Mongo_items as mongo
 import viewmodel as vm
 import os, requests, json
 from functools import wraps
+from loggly.handlers import HTTPSHandler
+from logging import Formatter, getLogger
+from pythonjsonlogger import jsonlogger
 
 def create_app():
     app = Flask(__name__)
     app.config.from_object('flask_config.Config')
-
+    app.logger.setLevel(app.config['LOG_LEVEL'])
+    if app.config['LOGGLY_TOKEN'] is not None:
+        handler = HTTPSHandler(f'https://logs-01.loggly.com/inputs/{app.config["LOGGLY_TOKEN"]}/tag/todo-app')
+        formatter = jsonlogger.JsonFormatter("[%(asctime)s] %(levelname)s in %(module)s: %(message)s")
```

Comment:
You've successfully added a JSON logger but haven't taken full advantage of structured logging. 

E.g. when you log `app.logger.debug("user id %s completed item id:%s", ...)`, that message is still just a single string. So you potentially need to use complicated queries, maybe including some regex, when you want to analyse your logs. E.g. find all logs about things user X has done, or for a specific action, or analyse usage by grouping logs by user id,

It would potentially be nicer if the log JSON contained properties like `"action": "COMPLETE_ITEM", "user_id": "...", "item_id": "..."`, depending on how you want to use the logs.

You do this with something like: `logger.info("your message", extra={"user_id": user_id})`. [See docs](https://github.com/madzak/python-json-logger#customizing-fields)